src/core/Akka.Streams/Dsl/Sink.cs

```diff
@@ -270,8 +270,7 @@ public static class Sink
         /// </summary>
         /// <typeparam name="TIn">TBD</typeparam>
         /// <returns>TBD</returns>
-        public static Sink<TIn, Task> Ignore<TIn>()
-            => new Sink<TIn, Task>(new SinkholeSink<TIn>(Shape<TIn>("BlackholeSink"), DefaultAttributes.IgnoreSink));
+        public static Sink<TIn, Task> Ignore<TIn>() => FromGraph(new IgnoreSink<TIn>());
```

Comment:
We don't return `Task<Done>>` here and I should write 
`RunWith(Sink.Ignore<int>(), Materializer) .PipeTo(TestActor, success: () => Done.Instance)` instead of `RunWith(Sink.Ignore<int>(), Materializer) .PipeTo(TestActor)`